site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -419,7 +425,7 @@ For example, consider the following directory structure:
 ```
 
 
-In this example we are creating an Email template and reading in the values for the text and html values from files in a sub-directory named emails. Reading files in like this allows you to format your emails nicely and Kickstart will handle the necessary JSON escaping to complete the API request.
+In this example you are creating an Email template and reading in the values for the text and html values from files in a subdirectory named `emails`. Using external files for templates like this allows you to format your emails nicely. Kickstart will handle the necessary JSON escaping to complete the API request.
```

Comment:
I think it should be Email Template or email template.  I think the later makes the most sense.
```suggestion
In this example you are creating an email template and reading in the values for the text and html values from files in a subdirectory named `emails`. Using external files for templates like this allows you to format your emails nicely. Kickstart will handle the necessary JSON escaping to complete the API request.
```